server/helpers/passport.js

```diff
@@ -11,10 +11,11 @@ const jwtStrategy = passportjwt.Strategy;
 const { ExtractJwt } = passportjwt;
 
 const JWT_SECRET = process.env.SECRET_KEY;
```

Comment:
Should this be in caps too? Why don't you destructure it alongside the other variables?

Reply:
the name JWT_SECRET gives it a better meaning that SECRET_KEY. So destructuring it will mean i'll have to change the name in the .env and that might start breaking stuff, that's why i left it

Reply:
Unless of course I change it and update the sample env file so everyone will need to change theirs... they also have to be changed on travis and heroku

Reply:
You can actually destructure and rename on one line by adding "SECRET_KEY: JWT_SECRET"

Reply:
You get? or maybe I should change the var from jwt_secret to secret_key instead. My issue with this approach is that the name `secret_key` doesnt give it meaning

Reply:
All right. But I just think it's weird to have a bunch of destructured env variables and one standing on it's own.

Reply:
ok @emp-daisy .. that's really nice

Reply:
Check the new changes

Reply:
Just to confirm `const { SECRET_KEY: JWT_SECRET } = process.env.SECRET_KEY;` is it supposed to look like this or this `const { JWT_SECRET: SECRET_KEY } = process.env.SECRET_KEY;` @emp-daisy

Reply:
Nice